pubspec.yaml

```diff
@@ -1,5 +1,11 @@
 name: fluri
 version: 0.0.0
 description: 'Fluri is a fluent URI library built to make URI mutation easy'
+authors:
+  - Evan Weible <evan.weible@workiva.com>
+  - Max Peterson <maxwell.peterson@workiva.com>
+  - Trent Grover <trent.grover@workiva.com>
+documentation:
```

Comment:
Including this but leaving it blank will tell pub to generate the docs automatically and host them on pub.